db/migrate/20160209143102_add_foreign_travel_advice_subscriber_lists.rb

```diff
@@ -0,0 +1,18 @@
+require 'csv'
+
+class AddForeignTravelAdviceSubscriberLists < ActiveRecord::Migration
+  def change
+    csv_path = File.join(File.dirname(__FILE__), "20160209143102_add_foreign_travel_advice_subscriber_lists.csv")
+
+    CSV.foreach(csv_path) do |gov_delivery_id, country_slug|
+      SubscriberList.create!(
+        gov_delivery_id: gov_delivery_id,
+        links: {
+          countries: [country_slug]
```

Comment:
There's a problem here, not immediately but it'll bite us later.

The `links` field is intended to house content IDs rather than slugs.  Slugs are still supported in the (deprecated) `tags` field but that's a legacy thing intended to match against a `tags` field within the `details` hash.

The original ticket did say slugs but was a little vague about the details of these fields.  We should really be using content IDs now.

This will still work for the specific use case because the travel advice publisher controls the format of the strings sent direct to the API and nothing else needs to trigger it, but it will break once we switch it over to the email alert service via the message bus since the contents of the links hash won't match.